jwst/pipeline/level2association.py

```diff
@@ -0,0 +1,100 @@
+"""Open a Level2 association given various inputs"""
```

Comment:
Would it perhaps make more sense to have this module live within /associations? It's currently used by pipelines that live here, but theoretically it could be used by anyone, anywhere.